geode-core/src/test/java/org/apache/geode/internal/cache/tx/PartitionedTXRegionStubTest.java

```diff
@@ -127,6 +129,36 @@ public void destroyExistingEntryThrowsTransactionDataNodeHasDepartedExceptionIfI
     verify(stub, never()).trackBucketForTx(keyInfo);
   }
 
+  @Test
+  public void getEntryForIteratorReturnsEntryGotFromLocal() throws Exception {
```

Comment:
getEntryForIteratorReturnsEntryGotFromLocal is misleading here. Possibly getEntryForIteratorReturnsEntryGotFromTransactionHost is more appropriate.